docs/core-concepts/core-sdk/cli/dotnet-new.md

```diff
@@ -0,0 +1,44 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new .NET Core project
+
+## SYNOPSIS
+dotnet-new [--type] [--lang]
+
+## DESCRIPTION
+The new command provides a convenient way to initalize a valid .NET Core project and sample source code to try out the CLI toolset. 
+
+This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the directory: 
+
+1. A sample "Hello World" program that exists in `Program.cs` ( or `Program.fs` ) file.
+2. A valid `project.json` file
+
+After this, the project is ready to be compiled and/or edited further. 
+
+## Options
+
+`-l`, `--lang [C##|F##]`
+
+Language of project. Defaults to `C##`. Also `csharp` ( `fsharp` ) or `cs` ( `fs` ) works.
```

Comment:
Also `csharp` ( `fsharp` ) or `cs` ( `fs` ) works. -> `csharp` ( `fsharp` ) and `cs` ( `fs` ) are also valid language options.